xrpl4j-crypto-parent/xrpl4j-crypto-core/src/main/java/org/xrpl/xrpl4j/crypto/core/keys/PublicKey.java

```diff
@@ -13,16 +18,17 @@
 import org.xrpl.xrpl4j.codec.addresses.UnsignedByteArray;
 import org.xrpl.xrpl4j.codec.addresses.Version;
 import org.xrpl.xrpl4j.codec.addresses.VersionType;
-import org.xrpl.xrpl4j.crypto.core.signing.UnsignedByteArrayDeserializer;
-import org.xrpl.xrpl4j.crypto.core.signing.UnsignedByteArraySerializer;
+import org.xrpl.xrpl4j.crypto.core.keys.PublicKey.PublicKeyDeserializer;
+import org.xrpl.xrpl4j.crypto.core.keys.PublicKey.PublicKeySerializer;
 
+import java.io.IOException;
 import java.util.Objects;
 
 /**
  * A typed instance of an XRPL Seed, which can be decoded into an instance of {@link Decoded}.
  */
-@JsonSerialize(as = ImmutableDefaultPublicKey.class)
-@JsonDeserialize(as = ImmutableDefaultPublicKey.class)
+@JsonSerialize(using = PublicKeySerializer.class)
+@JsonDeserialize(using = PublicKeyDeserializer.class)
```

Comment:
Remind me -- why add the serializer in this annotation rather than in the Xrpl4j jackson module?